src/Spout/Writer/Common/Cell.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Box\Spout\Writer\Common;
+
+use Box\Spout\Writer\Common\Helper\CellHelper;
+
+class Cell
+{
+    /**
+     * Numeric cell type (whole numbers, fractional numbers, dates)
+     */
+    const TYPE_NUMERIC = 0;
+
+    /**
+     * String (text) cell type
+     */
+    const TYPE_STRING = 1;
+
+    /**
+     * Formula cell type
```

Comment:
Can you either add a comment saying that this is not used yet or remove it alltogether?